src/engine/controls/bpmcontrol.cpp

```diff
@@ -46,9 +46,9 @@ BpmControl::BpmControl(QString group,
     m_dSyncTargetBeatDistance.setValue(0.0);
     m_dUserOffset.setValue(0.0);
 
-    m_pPlayButton = new ControlProxy(group, "play", this);
-    m_pReverseButton = new ControlProxy(group, "reverse", this);
-    m_pRateRatio = new ControlProxy(group, "rate_ratio", this);
+    m_pPlayButton = std::make_unique<ControlProxy>(new ControlProxy(group, "play", this));
```

Comment:
Redundant new: `std::make_unique<ControlProxy>(group, "play", this)`

Reply:
Sorry I don't understand what do you mean with redundant.

Reply:
I guess he meant you don't need to call new there, so it's therefore redundant. You can just use: `std::make_unique<ControlProxy>(group, "play", this)`

Reply:
Ups. :-S Yes, now I see. Thanks!!

Reply:
Done